pandora/models/pandora.py

```diff
@@ -47,6 +47,7 @@ class PlaylistItem(PandoraModel):
     track_length = Field("trackLength", 0)
     track_token = Field("trackToken")
     audio_url = Field("audioUrl")
+    bitrate = Field("bitrate")
```

Comment:
Thank you for the merge. For some reason the ```bitrate``` field in ```PlaylistItem``` does not appear to have merged correctly.